.github/workflows/deploy.yml

```diff
@@ -8,9 +8,33 @@ on:
     branches: [dev, master]
 
 jobs:
+  check-diff:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      frontend: ${{ steps.check-diff.outputs.frontend }}
+      cloud-functions: ${{ steps.check-diff.outputs.cloud-functions }}
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
```

Comment:
Could remove the empty lines within jobs in this file too, for the sake of stylistic consistency between the two.